src/backend/api/meals.js

```diff
@@ -1,15 +1,150 @@
+// @ts-nocheck
+const { query } = require("express");
 const express = require("express");
+const { max, sum } = require("../database");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+
+
+router.get("/hh", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const rows = await knex('Meal').select('*');
+  
+    response.json(rows);
   } catch (error) {
     throw error;
   }
 });
+// 1.maxPrice	Number	Returns all meals that are cheaper than maxPrice.	api/meals?maxPrice=90
+router.get("/", async (request, response) => {
+  const queryString=request.query;
+    // 1.maxPrice
+    if("maxPrice" in queryString){
+      try{
+      const maxPrice = Number(queryString.maxPrice);
+      if(isNaN(maxPrice)){
+        response.send('maxPrice should be a number ')
+      }else{
+             mealsData = await knex("Meal").where("price", "<", maxPrice);
+         if(MaxPriceMeals.length !== 0){
+          response.send(mealsData);
```

Comment:
It is too early to send the response here because we also could have provided a `title` query parameter and here you are only responding with the meals that matched the price. And also when you later on line 64 try to send another response, you will see an error because you can only send 1 response.